README.md

```diff
@@ -140,15 +138,15 @@ Our project is showing it's age and migration across multiple Rails version. Her
 │   ├── datatables              # Admin area datatables
 │   ├── graphql                 # API
 │   ├── helpers
-│   ├── javascript              # This should be where all the js lives!
+│   ├── javascript              # Source JavaScript
 │   │   ├── controllers
 │   │   └── src
 │   ├── jobs                    # Importer logic - jobs are created by cron (`/lib/tasks`). There's a readme here with more info
 │   ├── mailers                 # Email configuration
 │   ├── models
 │   ├── policies                # Pundit rules for who can do and access what
 │   ├── uploaders               # CarrierWave rules for handling image and logo uploads
-│   ├── validators              # Postcode validator - should possibly live somewhere else
+│   ├── validators              # Postcode validator - should possibly live somewhere else, or have other validators moved in here
```

Comment:
comment: the only other place this kind of code would go (if just the postcode validator is in this folder) is in `app/helpers`

Reply:
there's a load of stuff here: https://github.com/geeksforsocialchange/PlaceCal/blob/repo-cleanup/app/models/concerns/validation.rb#L15

Reply:
Hmm... as I understand it, code in `app/concerns` would be separate from `app/validators`. If there's more than enough overlap, I would merge the two (just did a skim; can take a closer look tomorrow with fresher eyes). Otherwise, they can be left as is.

Reply:
yeah i think can be left as is for sure its just a bit of a violation of single responsibility principle but theres no hurry to change it